app/helpers/bootstrap_flash_helper.rb

```diff
@@ -0,0 +1,30 @@
+module BootstrapFlashHelper
+  ALERT_TYPES = [:success, :info, :warning, :danger] unless const_defined?(:ALERT_TYPES)
+
+  def bootstrap_flash(options = {})
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for bootstrap_flash is too high. [22.74/15] [:ref](http://c2.com/cgi/wiki?AbcMetric)
* Cyclomatic complexity for bootstrap_flash is too high. [7/6]
* Method has too many lines. [19/10] [:ref](https://github.com/bbatsov/ruby-style-guide#short-methods)